components/eamxx/docs/index.md

```diff
@@ -1,10 +1,11 @@
-# The C++ E3SM Atmosphere Model (EAMxx)
+# The E3SM Atmosphere Model in C++ (EAMxx)
 
-Some nice introductory text goes here! Maybe some figures, too. Who knows?!
+EAMxx is an entirely new atmosphere model for E3SM. It is written in C++ using the Kokkos performance portability library to enable it to run efficiently on CPUs, GPUs, and (hopefully) whatever comes next. Currently only the km-scale explicit-convection version called SCREAM (the Simple Cloud-Resolving E3SM Atmosphere Model) is available, but a low-resolution version is in the works.
 
-* The [User Guide](user/index.md) explains how to run EAMxx, both in
-  its standalone configuration and within E3SM.
-* The [Developer Guide](developer/index.md) contains all the information needed
-  to contribute to the development of EAMxx.
-* The [Technical Guide](technical/index.md) contains all the technical
-  information about EAMxx.
+Like the documentation for other component models, EAMxx documentation is divided into:
+
+* The [User Guide](user/index.md) which explains how to run EAMxx and describes all the compile-time and run-time options available for modifying a simulation
+* The [Developer Guide](developer/index.md) which contains all the information needed to contribute to the development of EAMxx
+* The [Technical Guide](technical/index.md) which describes the equations and numerical methods used by each of the specific process implementations and diagnostics available in the current version of EAMxx
```

Comment:
I think these could be less verbose:

* The [User Guide](user/index.md) - how to run EAMxx and options for modifying a simulation
* The [Developer Guide](developer/index.md) -  information needed to contribute to EAMxx development 
* The [Technical Guide](technical/index.md) - equations and numerical methods used in EAMxx

Also, should the technical guide really contain info about diagnostics? Seems like we should have a separate area for that. For the higher level E3SM docs we've been talking about dividing the guides into pre and post processing guides. EAMxx has special considerations for each - so maybe we will eventually want a set of notes here? idk

Or am I totally misunderstanding what "diagnostics" means and it's all about run-time diagnostics for debugging or online checks for numerical instability?

Reply:
Ok, I took you "less verbose" suggestions.

I think you are misunderstanding what diagnostics means. EAMxx has a "diagnostics" class that consists of variables you can ask to be written out which aren't calculated anywhere in the actual processes, but instead are assembled from other available field-managed variables. This is as opposed to diagnostics one would compute or look at by postprocessing data. Maybe we could come up with a better name for the diagnostics class?

Reply:
Since I took your suggestions for shorter bullets, diagnostics aren't mentioned here. In the actual tech doc where we describe them... we should be more clear. I guess diagnostic outputs is ok but I suspect people will still be confused. That's a then problem.